pset-functions.js

```diff
@@ -148,6 +262,27 @@ console.log('Problem 4:')
 
 // Add your code below this line
 
+function doubleLetters(word){
+  word = word.toLowerCase()
+  const letters = word.split("")
+  let counter = {}
+  let flag = false
+  letters.forEach((letter) => {
+    if(counter[letter] === undefined){
+      counter[letter] = 1
+    }
+    else {
+      counter[letter] += 1
+      flag = true
+    }
+  })
+  return flag
+}
+
```

Comment:
Katie, this one is close but not quite there. This doubleLetters function will return true so long as any letter appears twice; versus returning true if and only if the same letter appears consecutively. Please review a potential sample solution [here](https://github.com/jsd20200127/hw-02-functions-solution/blob/e87c29073cf7285a5045ec8708561f279474f2d2/pset-functions.js#L217-L236).